tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -289,8 +299,7 @@ def Do(
       for split_name, example_split in example_splits.items():
         (example_split
          | 'WriteSplit[{}]'.format(split_name) >> _WriteSplit(
-             artifact_utils.get_split_uri(output_dict[utils.EXAMPLES_KEY],
-                                          split_name)))
+             artifact_utils.get_split_uri([examples_artifact], split_name)))
```

Comment:
ah my bad, I thought we could use examples_artifact directly, if not, just keep it as it is (revert the change)

Reply:
no worries, i changed it back